values.yaml

```diff
@@ -123,6 +123,17 @@ namespaces:
   ingress:
     enabled: true
 
+# Proxy Node namespace for both the integration and production release
+- name: verify-eidas-proxy-node-deploy
+  owner: alphagov
+  repository: verify-proxy-node
+  branch: master
+  path: ci/build
```

Comment:
I think this path will need to be something different, otherwise the cluster will be trying to apply the current build pipeline into the new namespace.

We don't have a directory for it within the `ci` dir of the proxy node, but I'd suggests maybe `ci/deploy`?

Reply:
☝️ Yep, right now we have ci/integration and ci/prod which we'll no longer use, so ci/deploy makes sense to me.

I think that directory may need to exist to avoid the [verify deployer pipeline](https://cd.gds-reliability.engineering/teams/gsp/pipelines/verify-deployer) failing.